packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -177,14 +186,34 @@ export const azureDevOpsAllWorkItems = (
             provider as IntegrationProviderAzureDevOps
           )
 
-          const restResult = await manager.getAllUserWorkItems(null, null, false)
+          const restResult = await manager.getAllUserWorkItems(
+            queryString,
+            projectKeyFilters,
+            isWIQL
+          )
+
           const {error, workItems} = restResult
+
           if (error !== undefined || workItems === undefined) {
             console.log(error)
           } else {
-            returnWorkItems.push(...workItems)
+            returnedWorkItems.push(...workItems)
```

Comment:
+1 can we remove `returnedWorkItems` and loop through `workItems` on line 202?

Reply:
changed